### PR TITLE
Do not copy introspect.py file

### PR DESCRIPTION
**Pull request by AlanCoding**
Connect https://github.com/ansible/ansible-builder/issues/150

Seems to work after running just `TestPytz`

### CODE REVIEW

File: ansible_builder/main.py [hunk 1]
```diff
@@ -344,12 +345,6 @@ def create_folder_copy_files(self):
             dest = os.path.join(self.build_context, CONTEXT_BUILD_OUTPUTS_DIR, new_name)
             copy_file(requirement_path, dest)
 
-        # copy introspect.py file from source into build context
-        copy_file(
-            ansible_builder.introspect.__file__,
-            os.path.join(self.build_outputs_dir, 'introspect.py')
-        )
```
Comment (AlanCoding) [inline ansible_builder/main.py:351]: what would you say you do here

**Comment by shanemcd** [comment]
I'd like to test this when installed in the global site-packages directory to make sure we don't get a permission denied.

**Comment by AlanCoding** [comment]
Sure, if that gives us problems we can always put it in a throw-away tmp directory and still accomplish the main objective.

**Comment by AlanCoding** [comment]
I tested with the AWX container built for the `execution-environments` branch.

 - `docker exec -it --user=0 tools_awx_1 /bin/bash`
   - `git clone https://github.com/AlanCoding/ansible-builder.git`
   - `cd ansible-builder`
   - `git checkout alancoding/introspect_where_you_are`
   - `pip3 install .` --> `/usr/lib/python3.6/site-packages`
 - `docker exec -it tools_awx_1 /bin/bash`
   - `cd ansible-builder/test/data/pytz`
   - `ansible-builder build -f execution-environment.yml -t quay.io/alan/alan -c /tmp/ctx`
 
that worked fine

The permissions on the site packages:

```
bash-4.4# ls -la /usr/local/lib/python3.6/site-packages/ansible_builder
total 88
drwxr-xr-x. 3 root root  4096 Jan 15 01:56 .
drwxr-xr-x. 1 root root  4096 Jan 15 01:56 ..
-rw-r--r--. 1 root root  6317 Jan 15 01:55 cli.py
-rw-r--r--. 1 root root   157 Jan 15 01:55 colors.py
-rw-r--r--. 1 root root   693 Jan 15 01:55 constants.py
-rw-r--r--. 1 root root   271 Jan 15 01:55 exceptions.py
-rw-r--r--. 1 root root    22 Jan 15 01:55 __init__.py
-rw-r--r--. 1 root root  6828 Jan 15 01:55 introspect.py
-rw-r--r--. 1 root root 16757 Jan 15 01:56 main.py
drwxr-xr-x. 2 root root  4096 Jan 15 01:56 __pycache__
-rw-r--r--. 1 root root  2207 Jan 15 01:55 requirements.py
-rw-r--r--. 1 root root  2296 Jan 15 01:55 steps.py
-rw-r--r--. 1 root root  4386 Jan 15 01:56 utils.py
```

Just to check, I tried modifying the introspect.py script as the awx user and got permission denied.

Now I have not tested docker...

**Review by shanemcd — APPROVED** [review]
Amazing, thanks for this.

**Pull request merged**